models/issue.go

```diff
@@ -356,6 +358,83 @@ func (issue *Issue) GetIsRead(userID int64) error {
 	return nil
 }
 
+func (issue *Issue) AfterDelete() {
```

Comment:
I'm not sure about the AfterDelete() approach, it is only used in one place, but it sounded like a clean method to me.

Reply:
no it's not done that way ...

just look how repo deletion is done ;)